ui/pages/settings/index.scss

```
@@ -68,7 +68,7 @@
       }
 
       &__list {
-        background: var(--color-background-default);
+        background: var(--color-background-alternative);
```

```suggestion
        background: var(--color-background-default);
```

Should stay `--color-background-default` since the previous color was `white`